codedog/utils/email_utils.py

```
@@ -0,0 +1,158 @@
+    
+    Args:
+        to_emails: List of recipient email addresses
+        subject: Email subject
+        markdown_content: Report content in markdown format
+        cc_emails: List of CC email addresses
+            
+    Returns:
+        bool: True if email was sent successfully, False otherwise
+    """
+    # Check if email notification is enabled
+    if not env.get("EMAIL_ENABLED", "").lower() in ("true", "1", "yes"):
```

Ruff (0.8.2)

Test for membership should be `not in`

Convert to `not in`

(E713)